src/errors.ts

```diff
@@ -159,8 +159,8 @@ export function getAPIErrorBody(error: APIError): {[id: string]: any} {
 }
 
 export function apiErrorMiddleware(err: Error, req: Request, res: Response, next: NextFunction) {
-  Sentry.captureException(err);
   if (isAPIError(err)) {
+    Sentry.captureException(err);
```

Comment:
Do we not want to catch the rest?

Reply:
They should get caught by the fall through error handler.